src/main/resources/templates/company.html

```diff
@@ -4,69 +4,83 @@
       th:replace="layout :: layout(~{::title}, ~{::section})"
 >
 <head>
-    <title>Company Page - Pollima</title>
+    <title>Company - Pollima</title>
 </head>
 <body>
 <section>
 <main>
     <div class="container">
         <h1>Company Page</h1>
```

Comment:
Remove Page

Reply:
It's still there

Reply:
updated

Reply:
It's still there. Check the line number.